organize_flowcell.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+from glob import glob
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Organize flowcell for a specific project in /proj/ngi2016001/nobackup/NGI/DATA/<project>')
+    parser.add_argument('--runfolder', type=str, required=True, help='Name of runfolder (required)')
+    parser.add_argument('--project', type=str, required=True, help='Project name (required)')
+    parser.add_argument('--exclude_lane', type=str, required=False, default='', help='Lane number(s) to exclude (optional).')
+    parser.add_argument('--exclude_sample', type=str, required=False, default='', help='Sample name(s) to exclude (optional). Sample-1,Sample-2..')
+    parser.add_argument('--exclude_sampleID', type=str, required=False, default='', help='Sample ID(s) to exclude (optional). Sample-1,Sample-2..')
+    parser.add_argument('--suffix', type=str, required=False, help='Add suffix to output-folder (Project_suffix).')
+    return parser.parse_args()
+
+def get_sample_info(line, exclude_lane, exclude_sample, exclude_sampleID):
+    lane, sample_id, sample_name, *_, library_info = line.split(',')
+    library_name = library_info.split(':')[-1].strip()
+    include = not (lane in exclude_lane or sample_id in exclude_sampleID or sample_name in exclude_sample)
+    return lane, sample_id, sample_name, library_name, include
+
+def parse_samplesheet(samplesheet, project, exclude_lane, exclude_sample, exclude_sampleID):
+    sample_info = {}
+    with open(samplesheet) as fin:
+        for line in fin:
+            if project not in line:
+                continue
+            lane, sample_id, sample_name, library_name, include = get_sample_info(line, exclude_lane, exclude_sample, exclude_sampleID)
+            if include:
+                sample_info.setdefault(lane, {})[library_name] = [sample_id, sample_name]
```

Comment:
Just thinking out loud here: I think this works, since the same library shouldn't be run on the same lane with a different Sample ID. That wouldn't make any sense.

Reply:
The library name was not as unique as I though, so I restructured the dict to:
`sample_info.setdefault(sample_id, {})[sample_name] = [ lane, library_name]`